app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -1,14 +1,14 @@
 module ManageIQ::Providers
   module Openshift
     class ContainerManager::RefreshParser < ManageIQ::Providers::Kubernetes::ContainerManager::RefreshParser
-      def ems_inv_to_hashes(inventory)
+      def ems_inv_to_hashes(inventory, options = Config::Options.new)
```

Comment:
Forgot one thing: since you added options args to Kubernetes parser too, let's `super(inventory, options)` below.